src/style.css

```diff
@@ -1 +1,99 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  width: 100%;
+}
+
+header {
+  width: 100%;
+  padding-right: 50px;
+}
+
+.container__logo {
+  padding: 10px 0;
+  line-height: 0;
+  text-decoration: none;
+}
+
+.nav__a {
+  position: relative;
+  margin: 0 10px;
+  padding: 24px 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  line-height: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__li {
+  display: contents;
+  list-style-type: none;
+}
+
+.header__img {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;

```

Comment:
we will add a padding to header
```suggestion

```

Reply:
it doesn't work with padding instead of margin